pages/common/xo.md

```diff
@@ -0,0 +1,23 @@
+# xo
+
+> A pluggable zero configuration linting utility for JavaScript.
+
+- Lint files in `src`:
+
+`xo`
+
+- Lint on files:
```

Comment:
on _a given set of_ files

Reply:
I've used the `eslint` page as a template but okay, will change that.